oasp/my-thai-star
# Docker compose deployment (Angular client + Java server)

## Added docker-compose support to:
1. Build both client and server artifacts (using *node* and *maven* images)
2. Deploy artifacts in *nginx* and *tomcat* containers using a `nginx.conf` file to reverse-proxy requests from client to server.

## Way to test it:
1. Clone the fork: `git clone https://github.com/cbeldacap/my-thai-star.git`. You can use your own machine or login to https://labs.play-with-docker.com/
2. Execute `docker-compose up`
_It will surely take several minutes_
3. Go to your `https://localhost:8080` and find the My Thai Star client.
4. Open, for example, the "Menu" tab and check in "Networks" if the server path is `http://localhost:8080/api`
